client/src/components/reviews/AddReview.jsx

```diff
@@ -210,15 +65,18 @@ class AddReview extends React.Component {
     });
   }
 
-  rating (e) {
-    console.log(e.target.value);
+  rating (rating) {
     this.setState({
-      rating: e.target.value
+      rating: rating
     });
   }
 
   sendData (e) {
     e.preventDefault();
+    if (!this.formValidate()) {
+      alert('You must enter the following: Any mandatory fields are blank, The review body is less than 50 characters, The email address provided is not in correct email format or The images selected are invalid or unable to be uploaded');
```

Comment:
i wonder is alert here a way of debugging? or did you mean to actually have an alert window in your final product?